python-sdk/tests/benchmark/Makefile

```diff
@@ -59,7 +59,8 @@ run_job:
 	@kubectl apply -f infrastructure/kubernetes/namespace.yaml
 	@kubectl apply -f infrastructure/kubernetes/postgres.yaml
 	@kubectl apply -f infrastructure/kubernetes/postgres_svc.yaml
-	@kubectl create -f infrastructure/kubernetes/job.yaml
+	@sed s/%VERSION%/${GIT_HASH}/g infrastructure/kubernetes/job.yaml > infrastructure/kubernetes/job_final.yaml
+	@kubectl create -f infrastructure/kubernetes/job_final.yaml
```

Comment:
This looks like a hack to me. There has to be another way of passing values from outside into the yaml file.

Reply:
@feluelle This is not the ideal solution but is there an easier alternative? 

I considered `yq` to replace the yaml file but then again you have to create a new file and you need to install `yq`.

Reply:
There is another hack - https://stackoverflow.com/a/56240712 :)

Reply:
But it is just a hack -- you can't template k8s yaml without Helm natively I think

Reply:
Right - you would have to use helm.